docs/update-locking.md

```diff
@@ -17,6 +17,8 @@ On devices running Supervisor v13.1.0 or newer, the Supervisor creates lockfiles
 
 ### Location of the lockfile
 
+__Note:__ Please do not try to delete `/tmp/balena` or legacy `/tmp/resin` directories, as you will get an error complaining of a busy resource.
```

Comment:
Github Markdown also supports this format, but I'm not sure if it is supported by our docs backend

```suggestion
> [!NOTE]  
> Trying to remove`/tmp/balena` or legacy `/tmp/resin` directories will result in an `resource busy` error
```

Reply:
Just scanning through the [local mode](https://github.com/balena-io/docs/edit/master/pages/learn/develop/local-mode.md) docs, it looks like `__Note:__` is the preferred syntax